planner/core/errors.go

```diff
@@ -100,4 +100,6 @@ var (
 	ErrDifferentAsOf       = dbterror.ClassOptimizer.NewStd(mysql.ErrUnknown)
 	ErrAsOf                = dbterror.ClassOptimizer.NewStd(mysql.ErrUnknown)
 	ErrOptOnTemporaryTable = dbterror.ClassOptimizer.NewStd(mysql.ErrOptOnTemporaryTable)
+	// ErrPartitionNoTemporary returns when partition at temporary mode
+	ErrPartitionNoTemporary = dbterror.ClassOptimizer.NewStd(mysql.ErrPartitionNoTemporary)
```

Comment:
What's the error message before? E.g. create a global temporary table with partitions.

Reply:
it is this err, but if i refer to origin err, it would introduce import cycle